### ISSUE TITLE
A Tax Receipt for the City of Austin

### ISSUE THREAD

**Issue by jcdwyer1** [atxhack4change/2016-project-proposals#13]
## Short description of the idea (140 character tweet):

A website that provides Austinites with a customized property tax receipt showing how their money is spent by the city. 

## What is your project concept or idea? What challenge or opportunity will it solve?

Next to mobility, affordability is arguably the hottest topic facing our growing city. Affordability can mean many things, but often the debate focuses on the size and purposes of our City budget.

Whether you rent or own in Austin, you are paying property taxes to fund City services. Yet the uses of your money are buried under confusing names in [700-page](https://assets.austintexas.gov/budget/15-16/downloads/Vol1Approved_Final.pdf) budget documents, making it hard to develop an informed opinion that can contribute to the public debate. 

How much do you pay in property taxes? Is it a reasonable amount? What purposes does your money serve? Are these costs going up or down?

A simple, well-designed tax receipt (similar to the one [created by the White House](https://www.whitehouse.gov/2014-taxreceipt)) can help answer these questions. 
 
## Who will benefit from your project? Describe the humans at the center of the problem - who are they? 
![alt text](https://s3-us-west-2.amazonaws.com/assets.feedingtexas.org/images/inline/IMG_2652.JPG "Photo of my tax receipt")
Nearly every Austin resident is a property taxpayer. Most Austin residents are feeling the squeeze of our rising cost of living. It's easy to point fingers towards City Hall, but very few Austinites have a good understanding of how our tax dollars are really put to work.

Users of this website will benefit from a greater understanding of how much their property taxes contribute to their overall housing costs, what those taxes pay for, and how much of their tax bill goes to which City services. 

Ideally the site will also give users the opportunity to share their new knowledge via social media, or act on it by contacting their City Council Member. 

## Issue area(s) relevant to your project idea: 

- Affordability
- Economic Development
- Economic Opportunity
- Open Government/Transparency

## What is the current state of your project idea?
- I have an idea or **concept** of a solution, but have not yet prototyped or tested it

## Tell us more about the current state of your project idea.

There are two public datasets that need to married by this website in order to provide a customized result:

1. The overall property taxes paid by each Austin address are [public and searchable](https://tax-office.traviscountytx.gov/properties/account-search). However, that data is in pdf form, and hard to parse if you don't know what you're looking at ([here's mine, for example](https://travis.go2gov.net/export?type=org&account=04142702580000)).To be successful, this project will need to figure out a way to use Travis County's property tax search essentially as a GIS service - users would input their address to our website, and our backend would query the Travis County website and extract/return the right number from the resulting pdf. Challenging! Maybe there's another way?
2. The share of each tax dollar funding various City services is detailed in the City of Austin's [annual budget documents](https://assets.austintexas.gov/budget/15-16/downloads/Vol1Approved_Final.pdf). However, that information is hard to find, and not formatted or phrased in a way that is helpful to the average Austinite. To be successful, we will need to extract the top-line spending categories from this document, divide them by the total City budget (general revenue) and use the resulting fractions as multipliers that will be applied to the resulting number from (1) above. I've already been working on this part, and expect to have it ready to go by June 3.

The resulting numbers generated by our website will show the exact amount that the property taxpayer contributed to each spending category. These numbers will then need to be graphically or textually presented in a way that is compelling and easy to understand. 

Ideally the page will also include functions for the user to share their result on social media (such as by share buttons) or take action by contacting their City Council Member. 

## Tell us about yourself. 

I participated in both previous ATX Hack for Change events as a project champion, and have a good idea of the right kind of scope & team skills needed for a fulfilling weekend project. 

In my day-job, I work for [Feeding Texas](http://www.feedingtexas.org), a statewide nonprofit leading a unified effort for a hunger-free Texas. In the last two hackathons I championed projects related to my work, both of which evolved into core functions of our website - our [SNAPshot data tool ](http://www.feedingtexas.org/learn/communities/snapshot-texas/)and [Texas Advocacy app](http://stage.feedingtexas.org/get-involved/advocate/) (currently undergoing maintenance). 

In my night-life, I help organize a free monthly barroom lecture series called [Nerd Nite](https://www.facebook.com/groups/338681696297530/). This means I'm good at talking about nerdy stuff (like web technology) without actually having much depth to my knowledge :)

This year, the project is a personal one. Based on the subject, you might think that I am anti-tax or anti-government, but I am actually quite the opposite. I think that most users of this proposed website will learn how *little* they actually pay towards City services, and how the relative share consumed by various City services is probably not what they expected. I'm hoping that this website will help our city *avoid* knee-jerk, anti-tax conversations once taxpayers are presented with information that is actually accurate and relevant to their lives.

**Comment by mateoclarke** [comment]
Hey @jcdwyer1! Glad to see you back as a project champion. Your proposal is very thorough. I've added some labels based on your response.

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by jcdwyer1** [comment]
JC Dwyer, jcdwyer@gmail.com

**Comment by danielhonker** [comment]
Consider your project accepted, @jcdwyer1! We invite you to come pitch it at the ATX Hack for Change.

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email in the next few days regarding your speaking slot for the Saturday morning pitches. Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by danielhonker** [comment]
BTW you might want to connect with @COAbudget. A team from the City's budget office is planning to pitch a project as well. [Here's their project](https://github.com/atxhack4change/2016-project-proposals/issues/35).

**Comment by danielhonker** [comment]
Consider your project accepted, @jcdwyer1. We invite you to pitch it at the ATX Hack for Change. You might also want to connect with @coabudget -- folks from the City's budget office will be there and plan to pitch a [project of their own](https://github.com/atxhack4change/2016-project-proposals/issues/35). 

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email in the next few days regarding your speaking slot for the Saturday morning pitches. Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by jcdwyer1** [comment]
Thanks, @danielhonker & @ssharif1 ! I'm excited to participate. 

@COAbudget, I'm available before the hackathon for coffee if you would like to compare notes.

**Comment by polunsky** [comment]
Congrats! If part of the goal is to show how low taxes are, you'd need a comparison - with similar cities, maybe. The Comptroller's site has a lot of data, that might work.

**Comment by jcdwyer1** [comment]
Repo now exists here: https://github.com/jcdwyer1/ATX-tax-receipt

**Comment by jcdwyer1** [comment]
@polunsky That sounds challenging. I remember how hard it was to find apples-to-apples comparisons when I was involved in a campaign to increase CoA library funding a couple years ago. Plus this project is only focused on the contribution of property taxes to the city pot; I'm sure other cities fund services in all manner of ways. Hopefully the raw number will be low enough? I know I was surprised.

**Comment by danielhonker** [comment]
@jcdwyer1 - you should have received an invite to join Slack. If you haven't already joined the team and seen the Slack channel for your project, it's #tax-receipt. We'll be sending out an email with details on pitches soon.

**Comment by jcdwyer1** [comment]
Hi all - This is pretty last minute, but unfortunately I won't be able to make the hackathon this weekend. I'm happy to re-open this proposal if someone else wants to work on it, but for now I'm closing it and will plan to resubmit next year. Happy hacking!
